i5k/settings.py

```diff
@@ -367,7 +367,7 @@
         },
         'i5k_file': {
             'class': 'logging.handlers.TimedRotatingFileHandler',
-            'filename': '/var/log/i5k/i5k.log',
```

Comment:
@ifishlin Why did you change this?  
Right now /var/log/django is framework level logging while i5k was supposed to be app level logging. 
Should we rethink the way this is set up?